forms-flow-web-root-config/sample.env

```diff
@@ -87,16 +87,18 @@ APPLICATION_NAME=formsflow.ai
 #To Enable Role level permission check for enabling Application
 ENABLE_APPLICATION_ACCESS_PERMISSION_CHECK=false
 
-#Custom theme URL
-#Used for providing theming configuration in a url that returns json format
-#  Example to change the theme a theme.json can be given with content:
+#CUSTOM_THEME_URL
+#You can provide custom theme by adding json data to following path
+#public/themeConfig/customTheme.json inside forms-flow-web-root-config.
+#the json data should be below format.
+
 #  `{
 #  "--navbar-background": "blue",
 #  "--navbar-items": "grey",
 #  "--navbar-active": "white"
 #  }`
 
-#CUSTOM_THEME_URL=
+#CUSTOM_THEME_URL=/themeConfig/customTheme.json
```

Comment:
I have one doubt. how users can get the variable names. and please add more description on sample.env to understand how they can override the theme

Reply:
okay

Reply:
There is some description added for sample.env now , later will add do documentation
![Uploading image.png…]()

Reply:
![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/61222674/aca0648c-d118-443e-977d-df8a18652f39)